app/src/main/java/org/fossasia/openevent/app/utils/ErrorUtils.java

```diff
@@ -75,6 +69,31 @@ public static String getErrorDetails(ResponseBody responseBody) {
         }
     }
 
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
+    public static String getErrorTitleAndDetails(Throwable throwable) {
```

Comment:
Rather return an object with relevant information set which can be reused for other purposes, then caller can reduce it to String if it wants